laravel/socialite/2015-07-23.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Insecure state generation
+link:      https://github.com/laravel/socialite/pull/91
```

Comment:
Yeh, that's what we thought @Ocramius. I'm still going to let Taylor know about this feedback though. :)